content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -133,507 +157,595 @@ types available by default:
-   
-.. image:: argentina/argentina_edi_14.png
-   :align: center
 
+The :guilabel:`PDF Report` related to electronic invoices that have been validated by the AFIP
+includes a barcode at the bottom of the format which represents the CAE number. The Expiration Date
+is also displayed as it's legal requirement.
+
+.. image:: argentina/invoice-printed-report.png
+   :align: center
+   :alt: Invoice printed report.
 
 Troubleshooting and Auditing
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-For auditing and troubleshooting purposes you can get the detailed information of an 
-invoice number that has been  previously sent to the AFIP,  
 
-.. image:: argentina/argentina_edi_23.png
+For auditing and troubleshooting purposes, it's possible to obtain detailed information of an
+invoice number that has been previously sent to the AFIP. To retreive this information, go into
+:doc:`Developer Mode <../../../../general/developer_mode>`, then go to the
+:menuselection:`Accounting` menu, and then click in the button :guilabel:`Consult Invoice` in AFIP.
+
+.. image:: argentina/consult-invoice-in-afip.png
    :align: center
-     
-.. image:: argentina/argentina_edi_24.png
-   :align: center  
+   :alt: Consult invoice in AFIP.
 
+.. image:: argentina/consult-invoice-in-afip-details.png
+   :align: center
+   :alt: Details of invoice consulted in AFIP.
 
-You can also get the last number used in AFIP for a specific Document Type and POS Number 
-as support for any possible  issues on the sequence synchronization between Odoo and AFIP. 
+It's also possible to retreive the last number used in AFIP for a specific Document Type and POS
+Number, as reference for any possible issues on the sequence synchronization between Odoo and AFIP.
 
-.. image:: argentina/argentina_edi_22.png
+.. image:: argentina/consult-last-invoice-number.png
    :align: center
-
+   :alt: Consult the last invoice number.
 
 Vendor Bills
 ------------
 
-Based on the purchase journal selected for the vendor bill, the document type is now a required field. 
-This value is auto populated based on the AFIP Responsibility type of Issuer and Customer, but the 
-value can be switched if necessary.
+Based on the purchase journal selected for the vendor bill, the document type is now a required
+field. This value is auto-populated based on the AFIP Responsibility type of Issuer and Customer,
+but the value can be switched if necessary.
 
-.. image:: argentina/argentina16.png
+.. image:: argentina/changing-journal-document-type.png
    :align: center
+   :alt: Changing journal and document type
 
-The document number needs to be registered manually and the format is validated automatically, in
-case that the format is invalid a user error will be displayed indicating the correct format that is
-expected.
+The document number needs to be registered manually and the format is validated automatically,
+however, in case the format is invalid a user error will be displayed indicating the correct format
+that is expected.
```

Comment:
Some small touchups here around tense and word choice.
```suggestion
The document number needs to be registered manually and the format will be validated automatically,
however, in case the format is invalid, a user error will pop up and indicate the correct format
that is expected.
```